### PR TITLE
development

### PR DESCRIPTION
**Pull request by dougd94**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
+*.sqlite3
 *.sqlite
+db.sqlite3
```
Comment (ignatiusukwuoma) [inline .gitignore:3]: Are you making use of sqlite as db?

File: accounts/forms.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    username = forms.CharField()
+    password = forms.CharField(widget=forms.PasswordInput)
+
+
+class UserRegistrationForm(UserCreationForm):
+    """Form used to register a new user"""
+
+    password1 = forms.CharField(
+        label="Password",
+        widget=forms.PasswordInput)
```
Comment (ignatiusukwuoma) [inline accounts/forms.py:19]: You should have a regex to match password with that ensures they contain certain group of characters

File: accounts/url_reset.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+from django.core.urlresolvers import reverse_lazy
+from django.contrib.auth.views import password_reset, password_reset_done, password_reset_confirm, password_reset_complete
+
+urlpatterns = [
```
Comment (ignatiusukwuoma) [inline accounts/url_reset.py:5]: What's the need of having these in a different file instead of urls.py

File: .gitignore (hunk 1 above)
Comment (dougd94) [reply .gitignore:3]: will switch to postgres on friday just trying to get the rest up to snuff

File: accounts/url_reset.py (hunk 3 above)
Comment (dougd94) [reply accounts/url_reset.py:5]: just following with one of the videos thats the way they did it haha

File: accounts/forms.py (hunk 2 above)
Comment (dougd94) [reply accounts/forms.py:19]: will look into it! thanks

**Pull request merged**